### PR TITLE
Fuseki optimizer and logging documentation improvements

### PR DESCRIPTION
**Pull request by eroux**
- document use of context keys `arq:logExec` and `arq:optReorderBGP`
- changing some `lg4j.properties` to `log4j2.properties`

### CODE REVIEW

File: source/documentation/tdb/optimizer.md [hunk 1]
```diff
@@ -65,18 +65,20 @@ Optimizer control files
 
 | File name   | Effect   |
 | ----------- | -------- |
-| `none.opt`  | No reordering - execute triple patterns in the order in the query |
```
Comment (afs) [inline source/documentation/tdb/optimizer.md:68]: Please leave `none.opt` in.

The issue is that there is reordering in the optimizer as well which helps put filter in better places. `none.opt` still applies at the TDB level.

**Review by afs — CHANGES_REQUESTED** [review]
The non.opt text should stay.

Otherwise, the PR looks good.

File: source/documentation/fuseki2/fuseki-logging.md [hunk 2]
```diff
@@ -53,6 +53,16 @@ which defaults to `/etc/fuseki` on Linux.
 For the standalone webapp server, `FUSEKI_BASE` defaults to directory `run/`
 within the directory where the server is run.
 
+The property `fuseki.loglogging` can also be set to `true` for additional logging.
```
Comment (OyvindLGjesdal) [inline source/documentation/fuseki2/fuseki-logging.md:56]: Would it be worth mentioning the fuseki environment variable `FUSEKI_LOGLOGGING also`, .ie.

The property `fuseki.loglogging` or the environment variable `FUSEKI_LOGLOGGING` can also be set to `true` for additional logging? 

I see it used in https://github.com/apache/jena/blob/main/jena-fuseki2/jena-fuseki-core/src/main/java/org/apache/jena/fuseki/system/FusekiLogging.java#L68

Comment (afs) [reply source/documentation/fuseki2/fuseki-logging.md:56]: @OyvindLGjesdal -- would you like to put in a PR?

The "meta" logging isn't the focus of this PR.

Merely FYI:
There is a companion setup in  `LogCtl`: `JENA_LOGLOGGING` `jena.loglogging`.

But Fuseki logging is more exposed to the runtime environment and a bit more complicated especially in war file form.

**Comment by eroux** [comment]
@afs thanks for the review! I've added `none.opt` back

**Review by afs — APPROVED** [review]

**Comment by afs** [comment]
Missed [the requested change](https://github.com/apache/jena-site/pull/120#issuecomment-1270001532) happening.

@eroux - Thanks!

**Pull request merged**